package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "eslint --ext .js,.ts . --cache --report-unused-disable-directives",
     "test:ci:cocoapods": "ruby packages/platform-ios/native_modules.rb",
     "postinstall": "yarn build",
-    "link-packages": "PACKAGES=('tools' 'platform-ios' 'platform-android' 'cli' 'cli-types' 'debugger-ui') && cd packages && for PACKAGE in \"${PACKAGES[@]}\"; do cd $PACKAGE && yarn link && cd ..; done",
+    "link-packages": "yarn workspaces run link-package",
```

Comment:
ok, so maybe we can go with a Node.js script and not use `workspaces run`?

Reply:
The 2 options right now are:

1. Change `global-ci/package.json` to be `exit 0;` and find a way to ignore it in lerna
1. Scratch the above and create a script that calls `yarn link` in all packages except `global-ci`

1 should be faster to get it done while 2 will take me a bit more time because of the other things that I'm working on.

I'm fine either way, just let me know what you prefer!

Reply:
I've pushed the change to use exit 0. I've tested it on Linux and Windows and looks like it's working.
@thymikee let me know if you prefer me to work on a node script instead of `workspaces run`